README.md

```diff
@@ -3,53 +3,77 @@
 ScratchCloud is a new, powerful, Scratch cloud data API for Node.js written by AmazingMech2418! 
 
 ## Features
-- Scratch login session handling
+- Scratch login session handling (no logouts)
 - Cloud data manipulation
 - Cloud data reading
 - Async/await options (using promises)
-- Ability to wait until a target variable is a certain value (async only)
+- Ability to wait until a target variable is less than, greater than, not equal, or equal to a value
+- Onload function to wait for a cloud variable to be loaded, for stability
+- Built in functions to mimic Scratch's cloud variable functions
+- Shows cloud changes that are NOT from your node program
 
 ## Examples
 
 ### Callbacks
 ```js
 // Require module
-const {Session, Cloud} = require('scratchcloud');
+const { Session, Cloud } = require('scratchcloud');
 
 // Create user session
+const project = 458027255 // Project ID
 const session = new Session(process.env.USERNAME, process.env.PASSWORD, function(user) {
   // Create cloud session
-  const cloud = new Cloud(user, 458027255, function(error, c) {
+  const cloud = new Cloud(user, project, function(error, cloud) {
     // On error, throw error
-    if(error)throw error;
+    if(error) throw error;
+    
+    // Wait until cloud variables have been loaded
+    cloud.onload("cloud");
```

Comment:
Only works for async.

Reply:
Technically, you *could* use it with callbacks.